lxqt-session/lxqt-config-session/man/lxqt-config-session.1
.TH lxqt-config-session "1" "2015-11-05" "LXQt 0.10.0" "LXQt Application/Environment Configuration"
.SH NAME
\fBlxqt-config-session\fR \- Application settings of \fBLXQt\fR: The Lightweight
Qt Desktop Environment
.SH SYNOPSIS
.B lxqt-config-session
.br
.SH DESCRIPTION
This application handles \fBLXQt\fR default applications settings, autostart
configuration, and environment settings.
.SH "REPORTING BUGS"
Report bugs to https://github.com/lxqt/lxqt/issues
.SH "SEE ALSO"
.\" any module must refer to the session application, for module overview and initiation
\fBstartlxqt.1\fR  LXQt session initialization and launch script (e.g. in \fB.xinitrc\fR)
.P
\fBlxqt-session.1\fR  LXQt \fIoverview\fR and complete session environment
.P
\fBlxqt-config.1\fR  LXQt settings Configuration Center interface
.P


CHANGELOG

lxqt-build-tools-0.6.0 / 2018-01-25
===================================

  * Bump version to 0.6.0
  * Don't use automatic string conversions
  * Introduced lxqt-transupdate for translation updates
  * Improved cmake scripting
    - Set cmake_minimum_required to 3.1.0
    - Consider QT_SELECT - some distributions use this tool
    - Drop 0X and 11 support, require c++14
    - Remvoed QUIET from any finds
    - Cleaned up LXQtTranslateTs module
    - Removed translation pulling and cleanup external repos
      after translations moved back to their original repos
    - Remove link-time optimization for gcc since it breaks
      qobject_cast() in some cases.
    - Removed obsolete FindFm module

lxqt-build-tools-0.5.0 / 2018-05-21
===================================

  * Bump version to 0.5.0
  * Prevent in-source builds
  * Adds an PreventInSourceBuilds module
  * http --> https
  * Fixed some LXDE mentions
  * LXQtCompilerSettings: Set CMAKE_C_VISIBILITY_PRESET to hidden
  * Don't allow Qt foreach/Q_FOREACH
  * Update CMakeLists.txt
  * Require Qt 5.7.1
  * Adds a FindXKBCommon CMake module
  * Adds ECMFindModuleHelpers CMake module
  * Don't export github templates

0.4.0 / 2017-09-22
==================

  * Release 0.4.0: Update changelog
  * Bump version to 0.4.0
  * Added needed versions
  * remove version set from LXQtConfigVars.in
  * Move LXQt version to liblxqt
  * Fixes a FTBFS with superbuild mode
  * Adds a FindExif module
  * LXQtCompilerSettings: Fixes a typo in diagnostics-color
  * Adds (Ninja) colored diagnostics for GCC (>=4.9)
  * Enable strict iterators for debug builds

0.3.2 / 2017-01-14
==================

  * Release 0.3.2: Update changelog
  * Bump version to 0.3.2 - finally the build tools are arch: all (#15)
  * Warnings (#16)
  * Remove arched bits from version.cmake-file (#14)

0.3.1 / 2017-01-01
==================

  * Release 0.3.1: Update changelog
  * Bump LXQT_PATCH_VERSION and build tools version (#13)
  * 32 64 bit ness (#12)
  * Use "gcc-ar" and "gcc-ranlib" tools when building static libs with gcc link-time optimization. (#11)

0.3.0 / 2016-12-06
==================

  * Release 0.3.0: Update changelog
  * Adds an FindXCB CMake module
  * FindUDev: Add VERSION_STRING support
  * FindUDev: Set REQUIRED_VARS
  * FindUdev: Feed pkg_modules_modules results to find_library/path
  * FindUDev: Adds Feature Summary
  * Adds FindUDev module
  * Enables colored diagnostics for CLang/Ninja combination
  * Disable link-time optimization for clang++/llvm since after some testing it breaks QObject signal/slot sometimes. * Remove trailing spaces.
  * Use different link-time optimization compiler options for gcc and clang and move -Bsymbolic flags to linker flags.
  * Add more linker options to turn on linker-time optimizations for gcc & clang.
  * Set CMAKE_BUILD_TYPE to Release by default if it's value is not set.
  * FindGLIB: Improve gio-unix handling
  * Adds a FindGLIB CMake module
  * Adds a FindMenuCache CMake module
  * Adds a FindFm CMake module
  * move project name to top
  * Fixes the name of package config file name in superbuild mode

0.2.0 / 2016-10-30
==================

  * Release 0.2.0: Update changelog
  * Bump minor to 2 (#7)
  * Added Qt5Core to README.md (#6)
  * Adds an LXQtConfigVars module
  * Adds superbuild support
  * README.md: Update dependencies

0.1.0 / 2016-10-14
==================

  * Release 0.1.0: Update changelog
  * Adds LXQtCreatePortableHeaders module
  * Authors and license (#2)
  * Add README.md
  * Make lxqt-build-tools be usable

0.0.0 / 2016-09-30
==================

  * Remove cpack (#107)
  * Bump version ot 0.11.0 (#105)
  * Settings: Add "homemade" XDG_CONFIG_DIRS support
  * translations: Change default translations repository
  * Don't lock the screen when it's already locked.
  * LXQtTranslateTs.cmake: Enable PULL_TRANSLATIONS by default
  * LXQtTranslateTs.cmake: Use/update template in remote repo
  * .gitignore: Cleanup
  * build: Use external translations
  * translations: Fix external translations logic
  * translations: Add external translations support
  * Adds CMAKE_INSTALL_PREFIX to the LXQt Theme search paths
  * LXQtCompilerSettings: Add QT_USE_QSTRINGBUILDER to the compile definitions
  * TranslateDesktopTs: Adds UPDATE_OPTIONS options
  * TranslateDesktop: Remove lupdate absolute option
  * TranslateDesktop: Use -a switch in grep commands
  * Use CMAKE_INSTALL_FULL_DATAROOTDIR
  * Add LXQT_GRAPHICS_DIR directory definition
  * Adds FindXdgUserDirs CMake find_package module
  * Release 0.10.0
  * pluginTranslation: removed using QStringLiteral as it was causing SEGFAULT in static finalization time
  * pluginTranslation: add some symbol for referencing (for file to be not stripped out in static linking time)
  * Adds "automatic" plugin translations loader support
  * Rename LxQt to LXQt everywhere
  * Handles CMake policy CMP0063
  * Adds an lxqt_enable_target_exceptions() CMake function
  * Adds LXQtCompilerSettings CMake module
  * LXQtCreatePkgConfigFile.cmake: Makes install COMPONENT configurable
  * LXQtTranslateTs.cmake Adds install COMPONENT option
  * Adds Runtime install COMPONENT
  * Fixes an SuperBuild/NormalBuild FTBFS
  * CMake: Use CMAKE_AUTOUIC, drop qt5_wrap()
  * CMake: Quote variables
  * CMake: Removes unused variables
  * Removes LIB_INSTALL_DIR CACHE PATH variable
  * Use relative paths in install DESTINATION
  * Marks link libraries as PUBLIC
  * Use target_compile_definitions() instead of add_definitions()
  * Make Release the default build type
  * Simplifies build type matching
  * lxqt-config.cmake.in: Use find_dependency()
  * Updates the build system to the Targets infrastructure
  * Sort .desktop translations. Fix typos in the comments. Remove local dir artifact.
  * pkg-config: Adds Qt5Xdg to the required dependencies
  * Updates lxqt_create_pkgconfig_file()
  * Stops using Qt5Xdg dropped stuff
  * Adds LXQT_DATA_DIR configuration variable

lxqt-session/lxqt-config-session/translations/lxqt-config-session.ts
<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE TS>
<TS version="2.1">
<context>
    <name>AutoStartEdit</name>
    <message>
        <location filename="../autostartedit.ui" line="14"/>
        <source>Edit</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../autostartedit.ui" line="20"/>
        <source>Name:</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../autostartedit.ui" line="27"/>
        <source>Search...</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../autostartedit.ui" line="47"/>
        <source>Command:</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../autostartedit.ui" line="57"/>
        <source>Wait for system tray</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../autostartedit.cpp" line="61"/>
        <source>Select Application</source>
        <translation type="unfinished"></translation>
    </message>
</context>
<context>
    <name>AutoStartItemModel</name>
    <message>
        <location filename="../autostartmodel.cpp" line="193"/>
        <source>Global Autostart</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../autostartmodel.cpp" line="195"/>
        <source>LXQt Autostart</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../autostartmodel.cpp" line="213"/>
        <source>Location: %1</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../autostartmodel.cpp" line="215"/>
        <source>Overrides: %1</source>
        <translation type="unfinished"></translation>
    </message>
</context>
<context>
    <name>AutoStartPage</name>
    <message>
        <location filename="../autostartpage.ui" line="17"/>
        <source>Delete</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../autostartpage.ui" line="24"/>
        <source>Add</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../autostartpage.ui" line="31"/>
        <source>Edit</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../autostartpage.ui" line="44"/>
        <source>Application Autostart</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../autostartpage.cpp" line="143"/>
        <source>Error</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../autostartpage.cpp" line="143"/>
        <source>File &apos;%1&apos; already exists!</source>
        <translation type="unfinished"></translation>
    </message>
</context>
<context>
    <name>BasicSettings</name>
    <message>
        <location filename="../basicsettings.ui" line="23"/>
        <source>Basic Settings</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../basicsettings.ui" line="30"/>
        <source>Window Manager</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../basicsettings.ui" line="49"/>
        <source>Search...</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../basicsettings.ui" line="59"/>
        <source>LXQt Modules</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../basicsettings.ui" line="65"/>
        <source>Start</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../basicsettings.ui" line="72"/>
        <source>Stop</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../basicsettings.ui" line="111"/>
        <source>Leave Session</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../basicsettings.ui" line="117"/>
        <source>Ask for confirmation to leave session</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../basicsettings.ui" line="124"/>
        <source>Lock screen before suspending/hibernating</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../basicsettings.ui" line="131"/>
        <source>Suspend/hibernate after lock delay:</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../basicsettings.ui" line="144"/>
        <source> ms</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../basicsettings.cpp" line="156"/>
        <source>Select a window manager</source>
        <translation type="unfinished"></translation>
    </message>
</context>
<context>
    <name>DefaultApps</name>
    <message>
        <location filename="../defaultappspage.cpp" line="74"/>
        <source>Select a terminal emulator</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../defaultappspage.cpp" line="79"/>
        <source>Select a web browser</source>
        <translation type="unfinished"></translation>
    </message>
</context>
<context>
    <name>DefaultAppsPage</name>
    <message>
        <location filename="../defaultappspage.ui" line="23"/>
        <source>Default Applications</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../defaultappspage.ui" line="32"/>
        <source>Terminal Emulation:</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../defaultappspage.ui" line="52"/>
        <location filename="../defaultappspage.ui" line="79"/>
        <source>Search...</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../defaultappspage.ui" line="59"/>
        <source>Web Browser:</source>
        <translation type="unfinished"></translation>
    </message>
</context>
<context>
    <name>EnvironmentPage</name>
    <message>
        <location filename="../environmentpage.ui" line="17"/>
        <source>Add</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../environmentpage.ui" line="24"/>
        <source>Delete</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../environmentpage.ui" line="50"/>
        <source>Environment Variables (Advanced)</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../environmentpage.ui" line="61"/>
        <source>Variable Name</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../environmentpage.ui" line="66"/>
        <source>Value</source>
        <translation type="unfinished"></translation>
    </message>
</context>
<context>
    <name>ModuleModel</name>
    <message>
        <location filename="../modulemodel.cpp" line="87"/>
        <source>Running</source>
        <translation type="unfinished"></translation>
    </message>
</context>
<context>
    <name>SessionConfigWindow</name>
    <message>
        <location filename="../sessionconfigwindow.cpp" line="45"/>
        <source>LXQt Session Settings</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../sessionconfigwindow.cpp" line="48"/>
        <source>Basic Settings</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../sessionconfigwindow.cpp" line="54"/>
        <source>Default Applications</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../sessionconfigwindow.cpp" line="57"/>
        <source>User Directories</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../sessionconfigwindow.cpp" line="63"/>
        <source>Autostart</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../sessionconfigwindow.cpp" line="69"/>
        <source>Environment (Advanced)</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../sessionconfigwindow.cpp" line="95"/>
        <source>Session Restart Required</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../sessionconfigwindow.cpp" line="96"/>
        <source>Some settings will not take effect until the next log in.</source>
        <translation type="unfinished"></translation>
    </message>
</context>
<context>
    <name>UserLocationsPage</name>
    <message>
        <location filename="../userlocationspage.cpp" line="116"/>
        <source>Locations for Personal Files</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../userlocationspage.cpp" line="185"/>
        <source>LXQt Session Settings - User Directories</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../userlocationspage.cpp" line="186"/>
        <source>An error ocurred while applying the settings for the %1 location</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../userlocationspage.cpp" line="202"/>
        <source>Choose Location</source>
        <translation type="unfinished"></translation>
    </message>
</context>
<context>
    <name>UserLocationsPrivate</name>
    <message>
        <location filename="../userlocationspage.cpp" line="63"/>
        <source>Desktop</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../userlocationspage.cpp" line="64"/>
        <source>Downloads</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../userlocationspage.cpp" line="65"/>
        <source>Templates</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../userlocationspage.cpp" line="66"/>
        <source>Public Share</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../userlocationspage.cpp" line="67"/>
        <source>Documents</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../userlocationspage.cpp" line="68"/>
        <source>Music</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../userlocationspage.cpp" line="69"/>
        <source>Pictures</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../userlocationspage.cpp" line="70"/>
        <source>Videos</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../userlocationspage.cpp" line="73"/>
        <source>Contains all the files which you see on your desktop</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../userlocationspage.cpp" line="74"/>
        <source>Default folder to save your downloaded files</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../userlocationspage.cpp" line="75"/>
        <source>Default folder to load or save templates from or to</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../userlocationspage.cpp" line="76"/>
        <source>Default folder to publicly share your files</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../userlocationspage.cpp" line="77"/>
        <source>Default folder to load or save documents from or to</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../userlocationspage.cpp" line="78"/>
        <source>Default folder to load or save music from or to</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../userlocationspage.cpp" line="79"/>
        <source>Default folder to load or save pictures from or to</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../userlocationspage.cpp" line="80"/>
        <source>Default folder to load or save videos from or to</source>
        <translation type="unfinished"></translation>
    </message>
</context>
</TS>
